www/index.php

```diff
@@ -377,6 +377,15 @@
                                         <input id="time" type="text" class="text short" name="time" value=""> seconds
                                     </li>
                                     <li>
+									<li>
+                                        <label for="time">

```

Comment:
Anything posted It won't take effect anyways unless you enable it in the .ini file, so it's safe. I'll still ask him tho